Packages/vcs/Lib/vcsvtk/isolinepipeline.py

```diff
@@ -27,6 +27,11 @@ def _updateVTKDataSet(self):
         data = vcs2vtk.numpy_to_vtk_wrapper(self._data1.filled(0.).flat,
                                             deep=False)
         self._vtkDataSet.GetPointData().SetScalars(data)
+        if (vcs2vtk._DEBUG_VTK):
+            writer = vtk.vtkXMLDataSetWriter()
+            writer.SetFileName("isoline-p.vts")
```

Comment:
@danlipsa I would suggest we take it out. in the past we have discussed leaving the debug code and I think I can understand that it makes it easy for developers to leave the debug code, it certainly add more lines and probably more maintenance to the code in the long run. Also, this will start other developers to follow the same and then we will have a lot of debug code. For example, we don't have this kind of code in VTK or ParaView. 

So I suggest we remove it now and discuss in more detail in the next meeting if that sounds reasonable to you.